src/main/java/nurseybook/model/task/Task.java

```diff
@@ -199,7 +130,7 @@ public boolean checkIfTaskRecurring() {
      * @return True if real task; false if ghost task.
      */
     public boolean checkIfRealTask() {
```

Comment:
Perhaps a better method name would be `isRealTask`? Similarly for the next method, maybe `fallsOnDate` would be better?

Reply:
Okay! I will update accordingly.